docker/osl.Dockerfile

```diff
@@ -3,11 +3,9 @@ ARG RUNTIME_IMAGE
 
 FROM ${BUILDER_IMAGE:-registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:1.35.0-6} AS builder
 
-ARG QUARKUS_EXTENSIONS
-ENV QUARKUS_EXTENSIONS=${QUARKUS_EXTENSIONS}
+ARG QUARKUS_EXTENSIONS=org.kie:kie-addons-quarkus-monitoring-sonataflow:10.0.0,org.kie:kogito-addons-quarkus-jobs-knative-eventing:10.0.0,org.kie:kie-addons-quarkus-persistence-jdbc:10.0.0,io.quarkus:quarkus-jdbc-postgresql:3.8.6.redhat-00004,io.quarkus:quarkus-agroal:3.8.6.redhat-00004
```

Comment:
if you remove that from here, also remove it from the build script

Reply:
here https://github.com/rhdhorchestrator/orchestrator-demo/blob/main/scripts/build.sh#L180-L191

Reply:
I totally missed that from the build script. I'll revert the change from the docker file, although I haven't seen the artifacts considered during the build. I'll run few tests to verify this.